TenElephants/TenElephants/Controllers/TrendPageController.swift

```diff
@@ -294,4 +311,19 @@ extension TrendPageController: UICollectionViewDataSource {
             return UICollectionReusableView()
         }
     }
+
+    func collectionView(_: UICollectionView, didSelectItemAt indexPath: IndexPath) {
+        guard let section = Section(rawValue: indexPath.section)
+        else { fatalError("No section provided") }
```

Comment:
Никаких fatalError-ов
```suggestion
        else { assertionFailure("No section provided") }
```